Fix build with meson 0.60
https://github.com/pwmt/zathura/commit/c6ec5044c5ff843979050bfb81742d14740bb656

This patch is included in 0.4.9 and should be removed when updating the port to
that version or later.
--- data/meson.build.orig
+++ data/meson.build
@@ -19,7 +19,7 @@ endif
 i18n = import('i18n')
 podir = join_paths(meson.project_source_root(), 'po')
 
-desktop = i18n.merge_file('desktop',
+desktop = i18n.merge_file(
   input: 'org.pwmt.zathura.desktop.in',
   output: 'org.pwmt.zathura.desktop',
   install: true,
@@ -28,7 +28,7 @@ desktop = i18n.merge_file('desktop',
   type: 'desktop'
 )
 
-appdata = i18n.merge_file('appdata',
+appdata = i18n.merge_file(
   input: 'org.pwmt.zathura.appdata.xml.in',
   output: 'org.pwmt.zathura.appdata.xml',
   install: true,
